### DISCUSSION TITLE
What does "bind" and "rejected" stand for in ClaimStatus?

### DISCUSSION THREAD

**Discussion by Sgitario** [halkyonio/primaza-poc#59] [Q&A] [answered]
The issue https://github.com/halkyonio/primaza-poc/issues/41 only mentions "new", "pending" and "error" statues, however the current Claim status has more statuses: https://github.com/halkyonio/primaza-poc/blob/main/servicebox-app/src/main/java/io/halkyon/services/ClaimStatus.java#L7

**Comment by cmoulliard** [comment, marked as answer]
As discussed yesterday, we should for the moment keep 3 status:
- New = when a claim is created
- Pending = Used to report to a user that the primaza application (job, ...) is currently checking if a service is available like credentials
- Bind = Secret has been created including url + credential

WDYT ? @Sgitario

**Reply by Sgitario** [reply]
Super

**Reply by Sgitario** [reply]
Plus, the Error status that is set when no service was found after several attempts.
